pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -53,7 +53,7 @@
 , firmware ? openSha256 != null || useFabricmanager
   # Whether the user accepts the NVIDIA Software License
 , config
-, acceptLicense ? config.nvidia.acceptLicense or false
+, acceptLicense ? config.nvidia.acceptLicense or true
```

Comment:
While this makes testing easier, I assume this is not intended to be committed?

Reply:
Agreed, this shouldn't be upstream to nixpkgs. The point is to ensure that people are "opting into" nvidia software license.

Reply:
I forget to delete it after tests in repl.